notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -591,23 +536,22 @@
         "\n",
         "! gcloud artifacts repositories create {DOCKER_REPOSITORY} \\\n",
         "    --repository-format=docker \\\n",
-        "    --location={REGION} \\\n",
+        "    --location={LOCATION} \\\n",
         "    --description=\"Repository for TensorBoard Custom Training Job\" \\\n",
         "    --quiet\n",
         "\n",
         "! gcloud artifacts repositories list"
       ]
     },
     {
-      "attachments": {},
       "cell_type": "markdown",
       "metadata": {
         "id": "MPqbvhO6tK_e"
       },
       "source": [
         "### Configure authentication to your private Docker repository\n",
         "\n",
-        "Before you push or pull container images, configure Docker to use the `gcloud` command-line tool to authenticate requests to `Artifact Registry` for your region."
+        "Before you push or pull container images, configure Docker to use the `gcloud` command-line tool to authenticate requests to `Artifact Registry` for your location."
```

Comment:
Artifact Registry references a product. Remove back ticks:

- `Artifact Registry` should change to: Artifact Registry

Reply:
Did the required changes